include/plat/arm/common/arm_def.h

```diff
@@ -10,7 +10,6 @@
 #include <common_def.h>
```

Comment:
As far as I can see, ``arm_def.h`` uses no definition from ``common_def.h`` so I think we could/should remove this inclusion.